metricbeat/docs/modules/elasticsearch.asciidoc

```diff
@@ -38,6 +38,9 @@ metricbeat.modules:
     #- ml_job
   period: 10s
   hosts: ["localhost:9200"]
+  #username: "elastic"
+  #password: "changeme"
+  #ssl.certificate_authorities: [ "/etc/pki/root/ca.pem" ]
```

Comment:
This file is autogenerated, to modify it you should modify the files in the module itself and then run `make update`. In this case the file you should modify is `metricbeat/module/elasticsearch/_meta/config.reference.yml`.

Reply:
I think having an example just makes it quicker to configure (especially since) those would be common options for Xpack users.

Reply:
Ok, lets add it. Could you add the change to `metricbeat/module/elasticsearch/_meta/config.reference.yml` and run `make update`?

Reply:
Sadly I do not know how to do that

Reply:
Oh, I see. You'd need git, go and make. Download the repository and run `make update` from the metricbeat repository. If you have any problem you cannot solve let us know :slightly_smiling_face:

Reply:
Other option is that I can continue with this PR if you want :slightly_smiling_face:

Reply:
if you have the time, and don't mind.

Reply:
Ok, we will take care of it. Thanks!